server/controllers/finance/reports/break_even_fee_center/index.js

```diff
@@ -0,0 +1,147 @@
+const q = require('q');
+const _ = require('lodash');
+const db = require('../../../../lib/db');
+const ReportManager = require('../../../../lib/ReportManager');
+
+const TEMPLATE = './server/controllers/finance/reports/break_even_fee_center/report.handlebars';
+const AccountReference = require('../../accounts').references;
+const setting = require('./setting');
+const getDistributionKey = require('../../distributionFeeCenter/getDistributionKey');
+
+// expose to the API
+exports.report = report;
+
+// default report parameters
+const DEFAULT_PARAMS = {
+  csvKey : 'brea_report',
+  filename : 'TREE.BREAK_EVEN_REPORT',
+  orientation : 'portrait',
+  footerRight : '[page] / [toPage]',
+};
+
+/**
+ * @function report
+ *
+ * @description
+ * This function renders the balance of accounts references as report.  The account_reference report provides a view
+ * of the balance of account_references for a given period of fiscal year.
+ */
+function report(req, res, next) {
+  const params = req.query;
+  const feeCentersChecked = [];
+
+  if (params.defineFeeCenters && params.feeCenters) {
+    params.feeCenters.forEach(item => {
+      if (parseInt(item, 10)) {
+        feeCentersChecked.push(item);
+      }
+    });
+  }
+
+  const data = {};
+  let reporting;
+
+  params.start_date = new Date(params.start_date);
+  params.end_date = new Date(params.end_date);
+
+  data.period = {
+    start_date : params.start_date,
```

Comment:
Why not just `start_date: new Date(params.start_date),` and then you won't need to the parsing above?

Reply:
I have set parsing in the line  40 and 41 before